### ISSUE TITLE
Training and testing accuracy avoided overfitting and have 98% validation accuracy but classification report is always 0.01%?

### ISSUE THREAD

**Issue by ganeshkuikel** [ganeshkuikel/fruit_classsification#1]
I have done fruit detection image classification problem using CNN i have done all the things upto training and fitting the model and my accuracy and validation accuracy are almost 100% but when i try to print classification report and confusion matrix from my model it always shows precision, recall and final accuracy is always 0.01% and confusion matrix is also bizzare. Why is this happenning please help me. Code is available at code section. Thank you if anyone would help me.
